core/capabilities/testutils/backend.go

```diff
@@ -0,0 +1,118 @@
+package testutils
+
+import (
+	"context"
+	"encoding/json"
+	"math/big"
+	"testing"
+	"time"
+
+	"github.com/ethereum/go-ethereum/accounts/abi/bind"
+	"github.com/ethereum/go-ethereum/accounts/abi/bind/backends"
+	"github.com/ethereum/go-ethereum/core"
+	"github.com/ethereum/go-ethereum/eth/ethconfig"
+	"github.com/smartcontractkit/chainlink-common/pkg/types"
+	"github.com/test-go/testify/require"
+
+	"github.com/smartcontractkit/chainlink/v2/core/chains/evm/assets"
+	evmclient "github.com/smartcontractkit/chainlink/v2/core/chains/evm/client"
```

Comment:
I thought the main reason why we added this code here was that it was difficult to write integration test for EVM, since EVM relayer has not yet been LOOPified

Reply:
no, the relayer is loopified, and now that we no longer need the return type info the capability in order not to lose type information on the return value we are good to move it

Reply:
discussed offline: for now going to leave here to make integration testing of the capability easier until we have a way to do this in the capabilities repo

Reply:
Based on offline discussion, leaving this capability here, since we are not able to test this capability with EVM simulated backend and EVM ChainReader in the new repo